internal/formatter/encryption.go

```diff
@@ -25,7 +25,10 @@ import (
 
 const (
 	keyAliasHeader   = "Key Alias"
-	defaultCmkFormat = "table {{.Provider}}\t{{.KeyAlias}}\t{{.SecurityPrincipals}}"
+	resourceIdHeader = "Resource ID"
+	keyNameHeader    = "Key Name"
+	awsCmkFormat     = "table {{.Provider}}\t{{.KeyAlias}}\t{{.SecurityPrincipals}}"
+	gcpCmkFormat     = "table {{.Provider}}\t{{.KeyName}}\t{{.ResourceId}}"
```

Comment:
Why do we need a separate format for GCP? Isn't KeyAlias similar to KeyName and SecurityPrincipals broad enough to cover the resource id?

Reply:
I made it different for each cloud provider to keep it consistent with the cloud UI. But we can keep it same for simplicity.

Reply:
seems something not working according to the unit test
```
Provider   <no value>                             <no value>
```

Reply:
@arishta-yb let me know if you need help with the unit test.

Reply:
Pushed the changes, it is fixed now.
Please take a look @cdavid .